README.md

```diff
@@ -8,6 +8,9 @@ ssh_key_ensure and purge_keys.
 
 This module may be used with a simple `include ::ssh`
 
+The `ssh::config_entry` defined type may be used directly and is used to manage
+Host entries in a personal `~/.ssh/config` file.
+
```

Comment:
@Phil-Friderici We could use a param `ssh::config_entries` that is a hash that is used with create_resources for ssh::config_entry resources.

Reply:
sure, like in the types and other modules